apps/context.go

```diff
@@ -67,7 +67,7 @@ type UserAgentContext struct {
 	AppID AppID `json:"app_id"`
 
 	// Fully qualified original Location of the user action (if applicable),
-	// e.g. "/command/helloworld/send" or "/channel_header/send".
+	// e.g. "/command/myapp/send" or "/channel_header/send".
```

Comment:
generally trying to avoid "hello" and especially "hello.*world" references throughout the code, for style.